mitosheet/src/mito/components/endo/IndexHeaderContextMenu.tsx

```diff
@@ -20,11 +20,14 @@ export default function IndexHeaderContextMenu(props: {
     actions: Actions;
     closeOpenEditingPopups: (taskpanesToKeepIfOpen?: TaskpaneType[]) => void;
 }): JSX.Element {
+    const [ currentSubmenu, setCurrentSubmenu ] = React.useState<string | undefined>(undefined);
+
     return (
         <Dropdown
             display={props.display}
             closeDropdown={() => props.setUIState((prevUIState) => {
                 const isCurrOpenDropdown = isCurrOpenDropdownForCell(prevUIState, props.rowIndex, -1);
+                setCurrentSubmenu(undefined);
```

Comment:
Oh -- we do close it here?

I think perhaps this function doesn't get close if another dropdown is open -- no dropdown is closed, only a new one is open. 

Maybe we can remove this call, and just use an effect on props.display that sets the submenu as undefined when the dropdown is closed.